src/components/Search/index.tsx

```diff
@@ -170,15 +168,10 @@ function Search({queryJSON, policyIDs, isCustomQuery}: SearchProps) {
     const sortedData = SearchUtils.getSortedSections(type, data, sortBy, sortOrder);
 
     const onSortPress = (column: SearchColumnType, order: SortOrder) => {
-        const currentSearchParams = SearchUtils.getCurrentSearchParams();
-        const currentQueryJSON = SearchUtils.buildSearchQueryJSON(currentSearchParams.q, policyIDs);
```

Comment:
Note: I removed this because it was super redundant. We already have the parsed `queryJSON` object in this component. Instead of using it we were getting the params and parsing them once again which didn't add anything to this function.